readyapi4j-core/src/main/java/com/smartbear/readyapi4j/TestRecipe.java

```diff
@@ -4,18 +4,25 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.smartbear.readyapi.client.model.TestCase;
+import com.smartbear.readyapi4j.extractor.ExtractorData;
 
 public class TestRecipe {
     private final TestCase testCase;
+    private ExtractorData extractorData;
 
-    public TestRecipe(TestCase testCase) {
+    public TestRecipe(TestCase testCase, ExtractorData extractorData) {
```

Comment:
Obviously this should be done on master so that doesn't affect this merge, but maybe something for @veinSmart to think about?